en/developer_guide/smartAdmin.rst

```diff
@@ -0,0 +1,84 @@
+SmartAdmin
+==========
+
+`SmartAdmin`_ is an responsive admin theme which use Bootstrap realize by Myorange.
+
+Open Orchestra's back office uses the specific design of SmartAdmin
+and only some of these JavaScript components (JarvisWidget and SmartNotification).
```

Comment:
Ok the presentation is far better now :)
A last sentence could indicate why you're talking about that. Something like "This doc will help you to add new features in the Back Office and respect it's general design."